staging/src/k8s.io/apiserver/pkg/util/webhook/client.go

```diff
@@ -142,7 +158,9 @@ func (cm *ClientManager) HookClient(cc ClientConfig) (*rest.RESTClient, error) {
 		// Use http/1.1 instead of http/2.
 		// This is a workaround for http/2-enabled clients not load-balancing concurrent requests to multiple backends.
 		// See https://issue.k8s.io/75791 for details.
-		cfg.NextProtos = []string{"http/1.1"}
+		if forceHTTP1 {
+			cfg.NextProtos = []string{"http/1.1"}
+		}
```

Comment:
oh, I see, for me is easier to read the code top down, and not have to remember that something is going to mutate it later ... it is also less fragile

Reply:
I don't have strong preference on either way. But changed as you suggested.